docs/source/build-run-openvino.md

```diff
@@ -0,0 +1,169 @@
+
+4. **Build OpenVINO Backend**: Once the prerequisites are in place, run the `openvino_build.sh` script to start the build process, OpenVINO backend will be built under `cmake-openvino-out/backends/openvino/` as `libopenvino_backend.so`
+
+   ```bash
+   ./openvino_build.sh
+   ```
+
+## Build Instructions for Examples
+
+### AOT step:
+Refer to the [README.md](../../examples/openvino/aot/README.md) in the `executorch/examples/openvino/aot` folder for detailed instructions on exporting deep learning models from various model suites (TIMM, Torchvision, Hugging Face) to openvino backend using Executorch. Users can dynamically specify the model, input shape, and target device. 
+
+Below is an example to export a ResNet50 model from Torchvision model suite for CPU device with an input shape of `[1, 3, 256, 256]`
+
+```bash
+cd executorch/examples/openvino/aot
+python aot_openvino_compiler.py --suite torchvision --model resnet50 --input_shape "(1, 3, 256, 256)" --device CPU
+```
+The exported model will be saved as 'resnet50.pte' in the current directory.
+
+#### **Arguments**
+- **`--suite`** (required):  
+  Specifies the model suite to use.  
+  Supported values:
+  - `timm` (e.g., VGG16, ResNet50)
+  - `torchvision` (e.g., resnet18, mobilenet_v2)
+  - `huggingface` (e.g., bert-base-uncased)
+
+- **`--model`** (required):  
+  Name of the model to export.  
+  Examples:
+  - For `timm`: `vgg16`, `resnet50`
+  - For `torchvision`: `resnet18`, `mobilenet_v2`
+  - For `huggingface`: `bert-base-uncased`, `distilbert-base-uncased`
+
+- **`--input_shape`** (required):  
+  Input shape for the model. Provide this as a **list** or **tuple**.  
+  Examples:
+  - `[1, 3, 224, 224]` (Zsh users: wrap in quotes)
+  - `(1, 3, 224, 224)`
+
+- **`--device`** (optional):  
+  Target device for the compiled model. Default is `CPU`.  
+  Examples: `CPU`, `GPU`
+
+### Build C++ OpenVINO Examples
+Build the backend and the examples by executing the script:
+```bash
+./openvino_build_example.sh
+```
+The executable is saved in `<executorch_root>/cmake-openvino-out/examples/openvino/`
+
+Now, run the example using the executable generated in the above step. The executable requires a model file (`.pte` file generated in the aot step), number of inference iterations, and optional input/output paths.
+
+#### Command Syntax:
+
+```
+cd ../../cmake-openvino-out/examples/openvino
+
+./openvino_executor_runner \
+    --model_path=<path_to_model> \
+    --num_iter=<iterations> \
+    [--input_list_path=<path_to_input_list>] \
+    [--output_folder_path=<path_to_output_folder>]
+```
+#### Command-Line Arguments
+
+- `--model_path`: (Required) Path to the model serialized in `.pte` format.
+- `--num_iter`: (Optional) Number of times to run inference (default: 1).
+- `--input_list_path`: (Optional) Path to a file containing the list of raw input tensor files.
+- `--output_folder_path`: (Optional) Path to a folder where output tensor files will be saved.
```

Comment:
Same for these args. Probalby it is best to showcase one e2e example here and have users refer to the other readme for details